### PR TITLE
#171253752 adding feedback 3 to the UI

### PR DESCRIPTION
**Pull request by irfiacre**
#### What does this PR do?
  This PR adds modified files applying feedback3 on UI .
#### Description of Task to be completed?

- feedback on the navigation bar, account creation form, cashier view.
- signup validation

 
#### How should this be manually tested?

- when I upload this on GitHub.

#### What are relevant pivotal tracker stories.
https://www.pivotaltracker.com/story/show/171253752

#### screenshots
image

image

image

image

### CODE REVIEW

**Comment by adafia** [comment]
Good job @irfiacre, However the pages that just have a log out link in the navigation. I suggest you align it to the right of the navigation bar.

File: UI/css/openAcc.css [hunk 1]
```diff
@@ -102,4 +105,10 @@ select,option{
         grid-template-columns: repeat(1,1fr);
         padding: 10px 3%;
     }
+    .personal{
+        margin: 10px;
+        display: grid;
+        grid-gap: 10px;
+        grid-template-columns: repeat(1,1fr);
+    }
 }
```
Comment (adafia) [inline UI/css/openAcc.css]: Please fix this new line issue

**Review by adafia — CHANGES_REQUESTED** [review]
Your commit message suggests it's a chore, it should be a bug fix instead. Please make this change

File: UI/css/cashier.css [hunk 2]
```diff
@@ -155,5 +162,13 @@ select{
...
+        padding: 0vh 0px;
+    }
+    .category{
+        grid-template-columns: repeat(1,1fr);
+        padding: 3px;
+    }
+    .actions{
+        height: 100vh;
     }
 }
```
Comment (adafia) [inline UI/css/cashier.css]: Here is another new line issue. Please go through all your files and make sure you add a blank line at the bottom of each file.

File: UI/css/openAcc.css (hunk 1 above)
Comment (irfiacre) [reply UI/css/openAcc.css]: may you clarify the issue please?

**Review by adafia — APPROVED** [review]

**Pull request merged**